### PR TITLE
Code Refractor: Use code-style enforcement

### PR DESCRIPTION
**Pull request by fakabbir**
- [x] _init_.py (**This PR**)
- [x] arcfour.py (**This PR**)
- [x] ascii85.py (**This PR**)
- [x] ccitt.py (**This PR**)
- [ ] converter.py    
- [ ] glyphlist.py   
- [ ] jbig2.py      
- [ ] lzw.py       
- [ ] pdfdevice.py    
- [ ] pdfinterp.py  
- [ ] pdftypes.py   
- [ ] runlength.py
- [ ] encodingdb.py   
- [ ] high_level.py  
- [ ] latin_enc.py    
- [ ] pdfdocument.py  
- [ ] pdfpage.py    
- [ ] psparser.py*  
- [ ] settings.py
- [ ] cmapdb.py  
- [ ] fontmetrics.py  
- [ ] image.py      
- [ ] layout.py    
- [ ] pdfcolor.py  
- [ ] pdffont.py      
- [ ] pdfparser.py  
- [ ] rijndael.py   
- [ ] utils.py

### CODE REVIEW

**Comment by pietermarsman** [comment]
Are you using line-width 120?

**Comment by fakabbir** [comment]
> Are you using line-width 120?

No, I see that flake8's default is set to 79. Have used it for clipping the line width.

**Comment by pietermarsman** [comment]
Ah, I see. Do you have any preference for specific line lengths? I always find 80 characters on the short side of the spectrum, but I dont have any rationale for that.

**Comment by fakabbir** [comment]
For me the preference is 80 characters, reason being 80 char are visible on splitting terminal in half screen and is a default choice by other developers too.

**Comment by pietermarsman** [comment]
Yep, that's the classical argument. Lets go with 80 then.

**Comment by pietermarsman** [comment]
Hi @fakabbir, are you going to continu to work on this? I expected you to go through the whole repo but I just noticed that your PR description could also mean that this PR is just about those four files.

**Comment by fakabbir** [comment]
> Hi @fakabbir, are you going to continu to work on this? I expected you to go through the whole repo but I just noticed that your PR description could also mean that this PR is just about those four files.

Hi @pietermarsman For my past experience such huge changes of refactoring can bring errors which gets unnoticed due to huge changes list. In order to make it simple for the reviewers to review and allow others to participate at the same time, I have made the checklist. **I would be working on other files too.** In case anyone else is interested feel free to check the files in checklist and raise PR.

**Comment by pietermarsman** [comment]
Ok, then I will have a look at the files you've already changed.

File: pdfminer/ccitt.py [hunk 1]
```diff
@@ -454,8 +452,9 @@ def _flush_line(self):
         return
 
     def _do_vertical(self, dx):
-        #print '* vertical(%d): curpos=%r, color=%r' % (dx, self._curpos, self._color)
-        #print '  refline:', self._get_refline(self._curpos+1)
+        # print '* vertical(%d): curpos=%r, color=%r' % (dx, self._curpos,
+        # self._color)
+        # print '  refline:', self._get_refline(self._curpos+1)
```
Comment (pietermarsman) [inline pdfminer/ccitt.py]: These sort of comments should be removed or replaced by `logger.debug()` statements.

Comment (pietermarsman) [reply pdfminer/ccitt.py]: I've changed my mind (really quick ;). Let's keep the comments for now (in this PR) and solely focus on code-style enforcement.

**Comment by pietermarsman** [comment]
I've added flake8 testing to the travis-ci on the pep8 branch: [891](https://travis-ci.org/pdfminer/pdfminer.six/jobs/607820996) errors to go 😉

**Comment by pietermarsman** [comment]
@fakabbir, I'm continuing your effort now on the `pep8` branch. When you continue to work on this, make sure you start from that branch.

**Pull request merged**